### DISCUSSION TITLE
middleware/kafka/user-guide/insight/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1897] [Announcements]
# middleware/kafka/user-guide/insight/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/middleware/kafka/user-guide/insight/

**Comment by liyuerich** [comment]
实例监控的图都没有数据，能否换一张？

**Reply by windsonsea** [reply]
updated in PR #5302
